operator/internal/controller/pulumi/suite_test.go

```diff
@@ -64,7 +69,7 @@ var _ = BeforeSuite(func() {
 		// Note that you must have the required binaries setup under the bin directory to perform
 		// the tests directly. When we run make test it will be setup and used automatically.
 		BinaryAssetsDirectory: filepath.Join("..", "..", "..", "bin", "k8s",
-			fmt.Sprintf("1.29.0-%s-%s", runtime.GOOS, runtime.GOARCH)),
+			fmt.Sprintf("1.28.3-%s-%s", runtime.GOOS, runtime.GOARCH)),
```

Comment:
Why downgrade?

Reply:
The suite for the workspace controller uses `1.28.3` and I think the CI script knows that.  I guess I used a slightly different scaffolder for the stack controller.